internal/future/serving/testing/server.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testing
```

Comment:
Is the pattern to have common helpers in a testing folder? I am curious as we write tests for individual components (frontend / backend etc.,) how should we structure this? (This is a general question for what pattern we should follow?)

Reply:
The Go API uses it and so does Kubernetes.

https://github.com/kubernetes/kubernetes/search?q=%22package+testing%22&unscoped_q=%22package+testing%22

Reply:
Ok. I was more curious of the level at which this package exists - so will there be a testing package under each of frontend, backend, mmlogic and each example, harness etc.? Irrelevant to this PR - so lets take this discussion offline.